doc/src/driver/compile.rst

```diff
@@ -28,6 +28,10 @@ Enable implicit offload copying
 
 Disable fast math optimization
 
+.. option:: --exhaustive-tune
```

Comment:
What about calling this flag `--exhaustive-optimize`? I think that sounds better.